lib/structs/events/voip.cpp

```diff
@@ -1,9 +1,26 @@
+#include <cstdlib>
+#include <string>
+
 #include "mtx/events/voip.hpp"
 
 #include <nlohmann/json.hpp>
 
 using json = nlohmann::json;
 
+namespace {
+uint16_t
+version(const json &obj)
+{
+        auto v = obj.at("version");
+        try {
+                return v.get<uint16_t>();
+        } catch (const json::type_error &) {
+                return std::atoi(v.get<std::string>().c_str());
+        }
```

Comment:
I think we should treat the version as a string instead and convert this to:

```suggestion
        if (v.is_number())
                return "0";
        } else {
                return v.get<std::string>();
        }
```

Reply:
Can definitely do that. This was just a quick fix to get calls back working with Element. Thanks for is_number, that's.. a lot cleaner.